src/errors.rs

```diff
@@ -1,3 +1,5 @@
+static ERROR_PREFIX: &'static str = "CLI assertion failed";
```

Comment:
Oops, I tend to forget that there are consts too. ^^

I'll link it in a todo list, at the top comment. :-)